### PR TITLE
Add CORS headers

### PR DESCRIPTION
**Pull request by bhb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,11 +86,21 @@ You are welcome and encouraged to run the API locally on your machine. To do so,
 
 ### Dependencies
 
-* SBT (Scala Build Tool) &ndash; available via Homebrew
+* SBT 0.13.x (Scala Build Tool) &ndash; available via Homebrew
```
Comment (hrs) [inline README.md]: This is truly the smallest nit, and definitely feel free to ignore it, but an [en dash is for numeric ranges](https://typographyforlawyers.com/hyphens-and-dashes.html). We probably want an em dash here (`&mdash;`).

**Comment by bhb** [comment]
@hrs Thanks for the review! I didn't intend to merge this before fixing these, but I ended up being pressed for time. I'll open up a follow-up PR to fix these.

File: app/web/controllers/PetsController.scala [hunk 2]
```diff
@@ -32,6 +37,13 @@ class PetsController @Inject()(findPet: Find[Pet], createPet: Create[CreatePetFo
   def create = Action.async(parse.json) { request =>
     createEntity(request, createPet)
   }
+
+  def hello = Action {
+    val nowAsISO = ZonedDateTime.now( ZoneOffset.UTC ).format( DateTimeFormatter.ISO_INSTANT )
+
+
```
[the remark this answers is not in the document]
Comment (bhb) [reply app/web/controllers/PetsController.scala]: Good point! Will fix.

File: README.md (hunk 1 above)
Comment (bhb) [reply README.md]: Good eye, I'll fix this!

**Pull request merged**